docs/testing.md

```diff
@@ -40,95 +42,98 @@ Before running these commands, make sure you have Node.js installed. For instruc
-| `amp e2e --config=<config>`                              | Same as above, but `config` can be `prod` or `canary`. (Defaults to `prod`.)                                                                                                                                                                           |
-| `amp e2e --nobuild`                                      | Same as above, but skips building the runtime.                                                                                                                                                                                                         |
-| `amp e2e --files=<test-files-path-glob>`                 | Runs end-to-end tests from the specified files on the latest Chrome browser.                                                                                                                                                                           |
-| `amp e2e --testnames`                                    | Lists the name of each test being run, and prints a summary at the end.                                                                                                                                                                                |
-| `amp e2e --engine=ENGINE`                                | Runs end-to-end tests with the given Web Driver engine. Allowed values are `puppeteer` and `selenium`.                                                                                                                                                 |
-| `amp e2e --headless`                                     | Runs end-to-end tests in a headless browser instance.                                                                                                                                                                                                  |
-| `amp e2e --watch`                                        | Watches for changes in test files, runs tests.                                                                                                                                                                                                         |
-| `amp check-sourcemaps`                                   | Checks sourcemaps generated during minified compilation for correctness.                                                                                                                                                                               |
+| Command                                                   | Description                                                                                                                                                                                                                                            |
```

Comment:
Entire table is diffed due to formatting. Actual change is adding the `storybook` commands below.